var/spack/repos/builtin/packages/leveldb/package.py

```diff
@@ -42,12 +43,28 @@ def install(self, spec, prefix):
 
         mkdirp(prefix.include)
         mkdirp(prefix.lib)
+        mkdirp(join_path(prefix.lib, 'pkgconfig'))
 
         cp = which('cp')
 
         # cp --preserve=links libleveldb.* prefix/lib
-        args = glob.glob('libleveldb.*')
-        args.append(prefix + '/lib')
+        args = glob.glob('out-shared/libleveldb.*') \
+            + glob.glob('out-static/libleveldb.*')
+        args.append(prefix.lib)
         cp('--preserve=links', *args)
 
-        cp('-r', 'include/leveldb', prefix + '/include')
+        cp('-r', 'include/leveldb', prefix.include)
+
+        with open(join_path(prefix.lib, 'pkgconfig', 'leveldb.pc'), 'w') as f:
+            f.write('prefix={0}\n'.format(prefix))
+            f.write('exec_prefix=${prefix}\n')
+            f.write('libdir={0}\n'.format(prefix.lib))
+            f.write('includedir={0}\n'.format(prefix.include))
+            f.write('\n')
+            f.write('Name: leveldb\n')
+            f.write('Description: LevelDB is a fast key-value storage library'
+                    ' written at Google that provides an ordered mapping from'
+                    ' string keys to string values.\n')
```

Comment:
You can actually access the package docstring for this if you don't want to duplicate it:
http://stackoverflow.com/questions/713138/getting-the-docstring-from-a-function